engine/engine.go

```diff
@@ -135,12 +135,22 @@ func (e *Docker) Destroy(ctx context.Context, specv runtime.Spec) error {
 
 	// stop all containers
 	for _, step := range append(spec.Steps, spec.Internal...) {
-		e.client.ContainerKill(ctx, step.ID, "9")
+		if err := e.client.ContainerKill(ctx, step.ID, "9"); err != nil {
```

Comment:
```suggestion
		if err := e.client.ContainerKill(ctx, step.ID, "9"); err != nil && !client.IsErrNotFound(err) && !errdefs.IsConflict(err) {
```

Because some containers might not exist (because of skipped steps) or are not running (already terminated)